ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TipsLayout.kt

```diff
@@ -1,84 +1,157 @@
+                bottom = dimensionResource(R.dimen.tutorial_page_inset_bottom)
+            ).verticalScroll(rememberScrollState())
     ) {
+        Spacer(modifier = Modifier.weight(1f))
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Column() {
+                Spacer(modifier = Modifier.height(290.dp))
+                Spacer(modifier = Modifier.minLinesHeight(1, MaterialTheme.typography.titleLarge))
+                Spacer(modifier = Modifier.minLinesHeight(8, MaterialTheme.typography.bodyMedium))
+            }
+
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                val composition by rememberLottieComposition(
+
+                    LottieCompositionSpec.RawRes(anim)
+                )
+
+                val progress by animateLottieCompositionAsState(
+                    // pass the composition created above
+                    composition,
+
+                    // Iterates Forever
+                    iterations = LottieConstants.IterateForever,
+
+                    // pass isPlaying we created above,
+                    // changing isPlaying will recompose
+                    // Lottie and pause/play
+                    isPlaying = true,
+
+                    // pass speed we created above,
+                    // changing speed will increase Lottie
+                    speed = 1f,
+
+                    // this makes animation to restart
+                    // when paused and play
+                    // pass false to continue the animation
+                    // at which is was paused
+                    restartOnPlay = false
+
+                )
+                LottieAnimation(
+                    composition,
+                    { progress },
+                    modifier = Modifier
+                        .height(290.dp)
+                        .fillMaxWidth()
+                )
+
+                Text(
+                    text = stringResource(title),
+                    style = MaterialTheme.typography.titleLarge,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp)
+                )
+
+                Text(
+                    stringResource(body),
+                    style = MaterialTheme.typography.bodyMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp)
+                )
+
+                if (body2 != null) {
+
+                    Text(
+                        stringResource(body2),
+                        textAlign = TextAlign.Center,
+                        modifier = Modifier.padding(horizontal = 32.dp)
+                    )
+                }
+            }
+        }
+        Box(
+            modifier = Modifier.weight(1f).fillMaxWidth()
+        ) {
+            if (page == Page.TIPS_START) {
```

Comment:
It'd be nice if we move the conditional into the `onClick` callback and the `Text` node itself so that we don't have to duplicate the same Modifiers on both buttons.

Something like: 
```
Button(
    onClick = {
        if(page == Page.TIPS_START) onTutorialAction(R.id.action_tips_finish) else nextPage()
    }
) {
    Text(stringResource(if(page == Page.TIPS_START) R.string.tutorial_tips_action_start else R.string.tutorial_tips_action_continue))
}